src/TestableHttpClient/Response/FunctionResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace TestableHttpClient.Response;
+
+internal class FunctionResponse : IResponse
```

Comment:
Rename to ResponseBuilderResponse or something like that, this class should be deprecated